packages/deskpro-messenger/src/components/chat/MessageForm.jsx

```diff
@@ -188,6 +192,10 @@ class MessageForm extends PureComponent {
     if(message !== this.state.message) {
       this.setState({ message }, () => this.handleTyping(message));
     }
+    if (this.wrapperRef.current.scrollHeight !== this.state.wrapperHeight) {
+      this.setState({ wrapperHeight: this.wrapperRef.current.scrollHeight });
+      this.props.scrollMessages();
```

Comment:
I would put scrollMessages call as a callback for setState, don't you think it's a good idea?